docs/resources/stream_processor.md

```diff
@@ -132,30 +132,30 @@ output "stream_processors_results" {
 ### Optional
 
 - `options` (Attributes) Optional configuration for the stream processor. (see [below for nested schema](#nestedatt--options))
-- `state` (String) The state of the stream processor. Used to start or stop the Stream Processor. Valid values are `CREATED`, `STARTED` or `STOPPED`.  When a Stream Processor is created without specifying the state, it will default to `CREATED` state.
+- `state` (String) The state of the stream processor. Commonly occurring states are 'CREATED', 'STARTED', 'STOPPED' and 'FAILED'. Used to start or stop the Stream Processor. Valid values are `CREATED`, `STARTED` or `STOPPED`. When a Stream Processor is created without specifying the state, it will default to `CREATED` state.
```

Comment:
sounds a bit repetitive to name the states twice (except FAILED).  maybe we can say something like: 
```
(String) The state of the stream processor. Valid values are `CREATED`, `STARTED` or `STOPPED`.  It can also be 'FAILED' if there is any problem. . Used to start or stop the Stream Processor. When a Stream Processor is created without specifying the state, it will default to `CREATED` state.
```

Reply:
I think it's worth it to repeat it because the "Commonly occurring states" and "valid values" are different sets of values so I think it's worth mentioning even if it's a bit repetitive